app/Models/Beatmapset.php

```diff
@@ -814,6 +814,26 @@ public function approver()
         return $this->belongsTo("App\Models\User", 'user_id', 'approvedby_id');
     }
 
+    public function userRatings()
+    {
+        return $this->hasMany(UserBeatmapsetRating::class);
+    }
+
+    public function ratingsCount()
```

Comment:
Also, should probably use a groupBy so it doesn't have to load the entire rating dataset into memory...